### PR TITLE
Google Fonts picker - remove need for google api

### PR DESCRIPTION
**Pull request by SaraVieira**
Google api was needed because of the display of the font in his style and with that removed there is no need to use an api key

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3050/commits/f7616a83fd35a31a9fc5f0e99fe69368f14522de is at https://pr3050.build.csb.dev/s/new.

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/AddFont/FontPicker/elements.ts [hunk 1]
```diff
@@ -1,6 +1,11 @@
-import styled, { css } from 'styled-components';
+import styled from 'styled-components';
 import Color from 'color';
 
+const svg = props =>
```
Comment (siddharthkp) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/AddFont/FontPicker/elements.ts]: For later: oh noes, would be cool to drop this in a file so it's discoverable?

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/AddFont/FontPicker/elements.ts [hunk 2]
```diff
@@ -105,23 +105,18 @@ export const SelectedFont = styled.button<{ done?: boolean }>`
...
-        height: 4px;
-        position: absolute;
-        right: 0.5rem;
-        top: 50%;
-        transform: translateY(-50%);
-      }
-    `}
+  :after {
+      content: '';
+      background-image: url("${props => svg(props)}");
```
Comment (siddharthkp) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/AddFont/FontPicker/elements.ts]: cheeky!

**Comment by siddharthkp** [comment]
`build-deps` didn't like the dependencies change, other than that, it's good!

**Comment by SaraVieira** [comment]
Yeah! Something is wrong with the build, I am gonna talk to bogdan

**Comment by CompuIves** [comment]
Merge with master should fix it!

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/AddFont/FontPicker/elements.ts (hunk 2 above)
Comment (SaraVieira) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/AddFont/FontPicker/elements.ts]: Thats how I roll

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/AddFont/FontPicker/elements.ts (hunk 1 above)
Comment (SaraVieira) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/AddFont/FontPicker/elements.ts]: Honestly I don't think we will ever used this again, I could have done in css now that I think of it

**Pull request merged**